core/modules/drr.h

```diff
@@ -237,7 +220,8 @@ class DRR final : public Module {
 
   // state map used to reunite packets with their flow
   CuckooMap<FlowId, Flow*, Hash, EqualTo> flows_;
-  llring* flow_ring_;   // llring used for round robin.
+  LockLessQueue<Flow*>* flow_ring_;   // queue used for round robin.
```

Comment:
While it is not part of this PR, I am not quite convinced that this module needs `LockLessQueue` to maintain a list of flows. All operations are single-item enqueue/dequeues so it would not give you any performance advantage (actually it will hurt). I'd rather recommend using `std::deque` to keep the code less foreign.